src/server/db/mongo/models/constants.js

```diff
@@ -1,4 +1,4 @@
 import config from '../../../server.config'
 export const MODEL_USER_PRIVATE = 'userprivate_' + (process.env.APP_NAME || config.network)
 export const MODEL_WALLET_NONCE = 'walletnonce'
-export const MODEL_PROPERTIES = 'server_properties'
+export const MODEL_PROPERTIES = 'server_properties_' + (process.env.APP_NAME || config.network)
```

Comment:
I've introduced envId constant could be re-used as collection suffix:

https://github.com/GoodDollar/GoodServer/pull/177/files#diff-04ee16c15e21801bb143c6e27d411ef2R3

Reply:
ok. next time these changes you can edit directly here in github ("insert suggestion" +- button in the comment editor)